lib/iex/test/iex/autocomplete_test.exs

```diff
@@ -86,6 +86,56 @@ defmodule IEx.AutocompleteTest do
     :code.delete(Sample)
   end
 
+  describe "Elixir no completion for default argument functions with doc set to false" do
+    test "real Elixir case" do
```

Comment:
It is just a matter of test for this test to fail and be removed, so please let's stick only with the test below and get rid of the describe.